src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/MSBuildRestoreUtility.cs

```diff
@@ -424,6 +424,18 @@ public static RestoreLogMessage GetWarningForUnsupportedProject(string path)
             return message;
         }
 
+        /// <summary>
+        /// Log warning NU1503
+        /// </summary>
+        public static RestoreLogMessage GetMessageForUnsupportedProject(string path)
+        {
+            var text = string.Format(CultureInfo.CurrentCulture, Strings.UnsupportedProject, path);
+            var message = new RestoreLogMessage(LogLevel.Minimal, NuGetLogCode.NU1503, text);
```

Comment:
I don't think this needs to be high verbosity.  Existing users who are already on static graph restore are already using it successfully so this would add noise.  This feature is really just for people onboarding to restore for the first time to help them figure out why a particular project isn't being restored.  If someone is onboarding and using minimal verbosity, they'll likely raise the verbosity or use the MSBuild binary logger to troubleshoot.

```suggestion
            var message = new RestoreLogMessage(LogLevel.Normal, NuGetLogCode.NU1503, text);
```

Reply:
Normal is not available, it will be `LogLevel.Information`?

Debug = 0,
Verbose = 1,
Information = 2,
Minimal = 3,
Warning = 4,
Error = 5

Reply:
Haha sorry yes, MSBuild has High, Normal, and Low, NuGet has Minimal, Information, and Verbose.  `LogLevel.Information` is my vote

Reply:
I find it unusual to have a warning code + minimal/information verbosity.
Most of our code assumes that warnings are warning/error level, so I'm wondering if there are any gotchas on that end.

Reply:
Good point, there's no point in associating the code as far as I know